tests/integ/test_git.py

```diff
@@ -126,46 +123,33 @@ def test_git_support_with_mxnet(sagemaker_local_session):
     with lock.lock(LOCK_PATH):
         try:
             serving_script_path = "mnist_hosting_with_custom_handlers.py"
-            client = sagemaker_local_session.sagemaker_client
-            desc = client.describe_training_job(TrainingJobName=mx.latest_training_job.name)
-            model_data = desc["ModelArtifacts"]["S3ModelArtifacts"]
-            model = MXNetModel(
-                model_data,
-                "SageMakerRole",
-                entry_point=serving_script_path,
-                source_dir=source_dir,
-                dependencies=dependencies,
-                py_version=PYTHON_VERSION,
-                sagemaker_session=sagemaker_local_session,
-                framework_version=MXNet.LATEST_VERSION,
-                git_config=git_config,
-            )
-            predictor = model.deploy(initial_instance_count=1, instance_type="local")
+            predictor = mx.deploy(1, "local", entry_point=serving_script_path)
+
             data = numpy.zeros(shape=(1, 1, 28, 28))
             result = predictor.predict(data)
             assert result is not None
         finally:
             predictor.delete_endpoint()
 
 
-@pytest.mark.skipif(PYTHON_VERSION != "py3", reason="Scikit-learn image supports only python 3.")
```

Comment:
Can we keep this here but commented out?  
Otherwise, if the tribal knowledge is lost, the dev that looks into fixing this might be confused, and may not think to look at git logs.

Reply:
I'll leave a comment in the code. I don't want to leave a commented-out annotation, though.

Reply:
Works for me =)